functions/Get-DBAQueryPlan.ps1

```diff
@@ -0,0 +1,93 @@
+﻿function Get-DbaQueryPlan {
+<#
+    .SYNOPSIS
+        Provides information about adhoc and prepared plan cache usage
+
+    .DESCRIPTION
+        "Checks ahoc and prepared plan cache for each database, if over 100 MBS should 
```

Comment:
Please make sentences all one line and do not add line breaks.

Reply:
You can also remove the double quotes and correct phrase `if over 100 MBs should consider you using...` to if over 100MBs you should consider using...`